modules/vectra.py

```
@@ -3,19 +3,35 @@
 import warnings
 import html
 import sys
+import re
 
 warnings.filterwarnings('always', '.*', PendingDeprecationWarning)
 
+class HTTPException(Exception):
+    def __init__(self,status_code, response):
+        try: 
+            r = response.json()
+            if 'detail' in r:
+                detail = r['detail']
+            elif 'errors' in r:
+                detail = r['errors'][0]['title']
+            elif '_meta' in r:
+                detail = r['_meta']['message']
+            else:
+                detail = response.content
+        except: 
```

Don't use bare except, at least use except Exception